Codeinwp/otter-blocks
# fix: sharing icons link to wrong post when used in neve custom layouts

Closes #1270.


### Summary

Find the post that needs to be shared from the current URL, rather than relying on `get_the_permalink()` to ensure the custom layout is not shared instead of the actual post.

### Test instructions

- Place a Sharing Icons block inside a Neve custom layout
- Add the custom layout to a post or page
- The buttons should share the link to the actual post/page and not to the custom layout
- Make sure it works with all types of permalink structure

Besides the custom layout context, they should also be checked in home pages, archive pages, widget areas. 



---- 

### Checklist before the final review
